blackbox/docs/sql/administration/user_management.txt

```diff
@@ -98,5 +98,12 @@ To list all existing users query that table::
 The column ``name`` shows the unique name of the user, the column ``superuser``
 shows whether the user has superuser privileges or not.
 
+.. warning::
+
+    When the :ref:`es_api_setting` is enabled, the ``sys.users`` table is
+    exposed. Therefor access to the users table is not only restricted to
+    superusers.
+
+
```

Comment:
i would it write differently as the sys.users table is not really exposed, it's just the data from the cluster state.

maybe: "... is enabled, it is possible to read the user data via the es api. Therefor the access is not restricted anymore." ? 

please also show it to @nslater